README.md

```diff
@@ -110,3 +110,56 @@ $ ./gradlew bintrayUpload # release embulk-input-sftp to Bintray maven repo
 ```
 $ ./gradlew test  # -t to watch change of files and rebuild continuously
 ```
+To run unit tests, we need to configure the following environment variables.
+
+When environment variables are not set, skip some test cases.
+
+```
+FTP_TEST_HOST
+FTP_TEST_USER
+FTP_TEST_PASSWORD
+FTP_TEST_SSL_TRUSTED_CA_CERT_FILE
+FTP_TEST_SSL_TRUSTED_CA_CERT_DATA
+```
+
+Following option is optional
+```
+FTP_TEST_PORT (default:21)
+FTP_TEST_SSL__PORT (default:990)
+```
+
+If you're using Mac OS X El Capitan and GUI Applications(IDE), like as follows.
+```xml
+
+launchctl setenv FTP_TEST_SSL_TRUSTED_CA_CERT_FILE$ vi ~/Library/LaunchAgents/environment.plist
```

Comment:
The `launchctl setenv ..` here is necessary?

Reply:
Fixed at 6de6352